client/src/bits_util.ts

```diff
@@ -1,9 +1,9 @@
-import { Bit, CHARACTER_MAP, LABEL_MAP, STAGE_MAP, CharacterId, StageId, LabelId } from './types';
+import { notEmpty, Bit, CHARACTER_MAP, LABEL_MAP, STAGE_MAP, CharacterId, StageId, LabelId } from './types';
 
 export const decorateBit = (bit:{ [key: string]: any}): Bit => new Bit({
     ...bit,
-    mainChars: bit.mainChars && bit.mainChars.map((char: CharacterId) => CHARACTER_MAP.get(char)),
-    vsChars: bit.vsChars && bit.vsChars.map((char: CharacterId) => CHARACTER_MAP.get(char)),
-    stages: bit.stages && bit.stages.map((stage: StageId) => STAGE_MAP.get(stage)),
-    standaloneTags: bit.standaloneTags && bit.standaloneTags.map((tag: LabelId) => LABEL_MAP.get(tag)),
+    mainChars: bit.mainChars && bit.mainChars.map((char: CharacterId) => CHARACTER_MAP.get(char)).filter(notEmpty),
```

Comment:
Either that or we could make our own function that filters out falsy values in prod, but throws an error in dev. If something gets filtered out here that means something went wrong and we should know about it.

Reply:
I was trying to spare your feelings by not going with filter(Boolean). But fine.

The second option is interesting but it would crash all the time right now because I haven't migrated the Bits in the DB yet. So maybe after that.